services/nginx/config/default.conf

```diff
@@ -25,6 +25,11 @@ server {
         return 302 /v3$request_uri;
     }
 
+    # redirect /<category>/<curie> to /<curie>
+    location ~^/[a-z]+/([^:]:[^:])$ {
+        return 302 /$2;
```

Comment:
This should probably also include the `absolute_redirect off;` directive like the "/api" block above, otherwise NGINX will attempt to construct an absolute URL and likely fail to produce a working one in non-production environments.